composer.json

```diff
@@ -30,12 +30,12 @@
         "ramsey/uuid": "^4.7.3"
     },
     "require-dev": {
-        "eventsauce/test-utilities": "dev-main",
+        "eventsauce/test-utilities": "^3.3",
```

Comment:
ℹ️ Support for PHPUnit 10 is now officially [available](https://github.com/EventSaucePHP/TestUtilities/releases/tag/3.3.0)